data_adapter_oemof/adapters.py

```diff
@@ -26,27 +27,40 @@ def as_dict(self):
         But we add custom_attributes (i.e. "name") which would be neglected.
         """
         fields = dataclasses.fields(self)
-        data = {field.name: getattr(self, field.name) for field in fields}
+        data = {}
+        for field in fields:
+            value = getattr(self, field.name)
+            data[field.name] = value
+            if isinstance(value, oemof.solph._plumbing._Sequence):
+                # if value.periodic_values:
+                #     data[field.name] = value.periodic_values
+                if len(value) != 0:
+                    data[field.name] = value.data
```

Comment:
This is not fail safe. If a `regular` sequence gets called once, it's length will be > 0

Reply:
Yes I am aiming to use the implementation that is commented above but since I did not switch to the "periodic-sequence-plumbing" yet since it did not seem settled to me.

Reply:
I do not see a Problem in saving the "real" sequence as we would have to save it somewhere and while moving it to timeseries data would be more percise the scalarss use is not inteded for such cases as far as I know.